pandas/plotting/_core.py

```diff
@@ -49,9 +49,13 @@ def _get_standard_kind(kind):
     return {'density': 'kde'}.get(kind, kind)
 
 
-def _gca():
+def _gca(figsize=None):
```

Comment:
I can do that. Note that I can't unconditionally overwrite figure.figsize, because if it's not specified, it's None, and that would overwrite the default in rcParams. That's why I have it pass in an empty dict when it's None.  Anyway, I can push that one level up the call stack like you mention.